### PR TITLE
media-sound/lollypop: Add me as maintainer, add 1.4.39, enable py3.12, fix shebang, print only relevant elogs, remove old

### PR DESCRIPTION
**Pull request by Leo3418**
Some other distributions have started to ship Lollypop 1.4.38 already (e.g. [Debian](https://salsa.debian.org/python-team/packages/lollypop/-/tree/debian/1.4.38-1?ref_type=tags), [Fedora](https://src.fedoraproject.org/rpms/lollypop/c/fd98d19ddd553d8e1ed7264464268c5b67c17b54?branch=rawhide), [openSUSE](https://build.opensuse.org/request/show/1164230)).  The upstream GitLab repo had a 1.4.38 tag a few weeks ago, but the tag has gone for unknown reason as of now, so we still have to use the `SRC_URI` on an unaffiliated domain (adishatz.org) instead of a GitLab tarball.  For this reason, I was a bit skeptical about adding 1.4.38 at first considering how a non-Git tarball could be added a backdoor in the recent xz-utils event; however, since Fedora is still using the same domain in their source URL, I would trust it for now.

### CODE REVIEW

File: media-sound/lollypop/lollypop-1.4.38.ebuild [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+PYTHON_REQ_USE="sqlite"
+inherit python-single-r1 gnome2-utils meson xdg
```
Comment (mgorny) [inline media-sound/lollypop/lollypop-1.4.38.ebuild:8]: Please keep inherits sorted.

File: media-sound/lollypop/lollypop-1.4.38.ebuild [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
```
Comment (mgorny) [inline media-sound/lollypop/lollypop-1.4.38.ebuild:6]: I suppose you want to use `Closes:` in the commit message to close the relevant bug.

File: media-sound/lollypop/lollypop-1.4.38.ebuild [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+		dev-util/desktop-file-utils
+	)
+"
+
+RDEPEND="
+	${DEPEND}
+	media-plugins/gst-plugins-pulse
+	app-crypt/libsecret[introspection]
+	dev-libs/totem-pl-parser[introspection]
+	gui-libs/libhandy:1[introspection]
```
Comment (mgorny) [inline media-sound/lollypop/lollypop-1.4.38.ebuild]: Please sort, it makes it easier to check things afterwards.

**Review by mgorny — CHANGES_REQUESTED** [review]

**Comment by Leo3418** [comment]
Upstream just released 1.4.39

**Comment by Leo3418** [comment]
> @Leo3418 with newly 1.4.39, I saw you still use tarball from adishatz, so is this your plan? and still works fine? (I think there would be some extra work if want to switch to use tag/release from upstream..)

Nice catch. I had added a brief comment above `SRC_URI` explaining why using tarballs from adishatz.org is necessary. The upstream Git repo uses a submodule, which is required to build the package but not included in GitLab tarballs. adishatz.org tarballs do include the submodules' files.

**Pull request closed without merging**